docs/emoji/charts-and-data.md

```diff
@@ -1,32 +1,51 @@
-# Emoji Charts and Data
+# Posting Emoji Charts and Data
 
-## Emoji beta charts
+The following use 15.0 as the example version. Change that to whatever the version is.
 
-always check and post the files that have changed in:
+## Posting Emoji beta charts
 
-<https://www.unicode.org/draft/emoji/>
+Check and post the chart files that have changed in:
 
-*   \*.html
-*   charts-BBBB/\* (except /internal/) - where BBBB is the beta release version
-    — in this case 12.0
-*   [future/](https://www.unicode.org/draft/emoji/future/)\* (except /internal/)
-*   [images/](https://www.unicode.org/draft/emoji/images/)\*
+https://github.com/unicode-org/emoji/tree/main/docs/emoji/
 
-In this case you should also post the following (just regenerated) since there
-are new images, and some changes in layout.
+That is:
 
-*   charts-RRRR/\* (except /internal/) - where RRRR is the current release
-    version — in this case 11.0
+*   First sanity-check the charts.
+    *   You can do a link check on the "flagged" charts, since they are built to be small.
+    *   Visually inspect them also; 
+        *   make sure that the new characters are in, 
+        *   version numbers are right,
+        *   and images are present (at least one per row)
+*   https://github.com/unicode-org/emoji/tree/main/docs/emoji/charts-15.0
+    *   All files with .html
+    *   Skip other files, and /internal/
+*   https://github.com/unicode-org/emoji/tree/main/docs/emoji/future
+    *   All files with .html
 
-(linking checking to be done with the "flag" versions. please let me know of any
-problems (though if there are no visible artifacts it isn't a showstopper))
+NOTE: make sure that all the emoji images are in a proper directory with proper names:
+*  **not** proposed/proposed_...png, but rather
+*  sample/sample_...png, or
+*  android/androic_...png, or
+*  ...
 
+[Not sure this comment is still relevant]
 First time posting a new set of beta charts, make sure that charts-beta is a
 newly created symlink to charts-BBBB.
 
-## Emoji beta data
-
-Copy all files under draft/Public/emoji/NN.N files to live Public/emoji/NN.NN
-*   don't copy anything under the "internal" directory.
-
-Check the readme file if it exists, which should say draft.
+## Posting Emoji beta data
+
+Copy all files under https://github.com/unicode-org/unicodetools/tree/main/unicodetools/data/emoji/15.0 files to Public
```

Comment:
```suggestion
Copy all files under https://github.com/unicode-org/unicodetools/tree/main/unicodetools/data/emoji/dev files to Public
```
Remember that we don't copy to .../data/category/15.0 until after the 15.0 release.

Reply:
I’m happy to help in Mark’s absence but am confused as to what needs to happen here. docs/build.md still says “We now generate files into an output folder with the Unicode version number” and #232 committed files to that directory, do you also want those files moved to match the proposed change?

Reply:
In the `Generated` output folder, we use versioned subfolders.
In the repo, the unreleased data files all go into `dev` subfolders. We have done this for all data files since we agreed last November to do so.
I can make a PR and shuffle these files.

Reply:
> I can make a PR and shuffle these files.

Actually, not for this pending PR. I can make a PR to move the files that got merged into the wrong place in PR #232.